### ISSUE TITLE
good work!

### ISSUE THREAD

**Issue by xinnie-the-pooh** [tebo1000/DS1220-replacement-PCB#1]
It's nice to see friends who do the same work here, interestingly, I boiled with vinyl chloride and added the chip and found that it was DS1210, DS2016, a PCB and battery, originally thought that it could end normally, but unfortunately evaporated too quickly, paste, where is the PCB inside to provide a schematic?
--translate by microsoft bing

**Comment by tebo1000** [comment]
I know this method but I went another way. Dallas/ Maxim provided aplication notes for the use of the DS1210. It is managing the CE (chip enable) signal in the event of a power loss. The coin cell battery is connected to the DS1210. I don`t have official schematics for the chip replacement but an example from the DS1210 datasheet and a sketch I made for the next chip I would replace, the DS1230. Hope it helps.
![example for DS1210](https://github.com/tebo1000/DS1220-replacement-PCB/assets/93089949/26ef0059-5db0-4da9-8c84-3dfd91a49cf0)
![DS1210 example](https://github.com/tebo1000/DS1220-replacement-PCB/assets/93089949/49a9847a-9742-4334-86da-55e11c1779c2)
